worker-taskunstowing/src/main/java/com/github/jobservice/workers/taskunstowing/TaskUnstowingWorkerFailure.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.jobservice.workers.taskunstowing;
+
+import com.google.common.base.MoreObjects;
+import com.hpe.caf.worker.document.model.Document;
+
+import java.text.MessageFormat;
+
+final class TaskUnstowingWorkerFailure
```

Comment:
Wouldn't it make more sense to make this an enumeration, so that you can keep the error codes and the corresponding descriptions together.  At the minute you have the descriptions spread throughout the code, which unfortunately encourages reusing error codes when new ones should be added.

See [ElasticWorkerError.java](https://github.houston.softwaregrp.net/caf/worker-elastic/blob/v1.4.0/worker-elastic-core/src/main/java/com/microfocus/caf/worker/elastic/core/ElasticWorkerError.java) for an example of doing this as an enumeration.